src/cljs/nr/utils.cljs

```diff
@@ -1,5 +1,6 @@
 (ns nr.utils
-  (:require [clojure.string :refer [join] :as s]))
+  (:require [clojure.string :refer [join, lower-case, split] :as s]
```

Comment:
Style thing: commas are considered white-space by Clojure, so we typically leave them out.

Reply:
Damn, I kept catching myself inserting unnecessary commas, good catch! Need to get a linter :D